mcda/mcda_with_robustness.py

```diff
@@ -87,8 +89,16 @@ def create_n_randomly_sampled_matrices(self) -> List[pd.DataFrame]:
         input_matrix = self._input_matrix  # (AxnI)
         is_exact_pdf_mask = self.is_exact_pdf_mask
         is_poisson_pdf_mask = self.is_poisson_pdf_mask
-
-        np.random.seed(42)
+        random_seed = self.random_seed
+
+        if random_seed is not None:
+            np.random.seed(random_seed)
+        else:
+            # TODO: the default_random_seed cannot be used while reading the settings from a JSON file, where None
+            #  cannot be given as an option; it will be implemented when the congiguration settings will be passed as a
+            #  stream or handle.
+            default_random_seed = 42
```

Comment:
I think you don't need an additional variable declaration for `42` I think it can directly be assigned as well. But however, you prefer.